docs/contributors/code/testing-overview.md

```diff
@@ -439,23 +439,23 @@ You can additionally have the devtools automatically open for interactive debugg
 npm run test-e2e:watch -- --puppeteer-devtools
 ```
 
-### Using alternate enviornment
+### Using alternate environment
 
-If you're using a different setup than wp-env, you can provide the base URL, username and password like this:
+If using a different setup than `wp-env`, you first need to symlink the e2e test plugins to your test site, from your site's plugins directory run:
 
 ```bash
-npm run test-e2e -- --wordpress-base-url=http://localhost:8888 --wordpress-username=admin --wordpress-password=password
+ln -s gutenberg/packages/e2e-tests/plugins/* .
 ```
 
-You also need to symlink all e2e test plugins to your site plugins directory:
+Then to run the tests, specify the base URL, username, and passwords for your site. For example, if your test site is at `http://wp.test`, use:
 
 ```bash
-ln -s gutenberg/packages/e2e-tests/plugins/* .
+WP_BASE_URL=http://wp.test npm run test-e2e -- --wordpress-username=admin --wordpress-password=password
 ```
 
 ### Scenario Testing
 
-If you find that end-to-end tests pass when run locally, but fail in Travis, you may be able to isolate a CPU- or netowrk-bound race condition by simulating a slow CPU or network:
+If you find that end-to-end tests pass when run locally, but fail in Travis, you may be able to isolate a CPU- or network-bound race condition by simulating a slow CPU or network:
```

Comment:
We should replace Travis with Github or Github Actions.

Reply:
Good catch :+1: